internal/config/migrate/from_1_17.go

```diff
@@ -60,11 +60,10 @@ func migrateFrom1_17(cfg *config.Config) error {
 	}
 
 	// Upgrade pause image
-	// https://github.com/cri-o/cri-o/pull/3582
-	logrus.Infof("Checking for pause_image, which now should be k8s.gcr.io/pause:3.2 instead of k8s.gcr.io/pause:3.1")
-	newPauseImage := "k8s.gcr.io/pause:3.2"
+	// https://github.com/cri-o/cri-o/pull/4550
+	logrus.Infof("Checking for pause_image, which now should be %s instead of k8s.gcr.io/pause:3.1", config.DefaultPauseImage)
 	if cfg.PauseImage == "k8s.gcr.io/pause:3.1" {
-		cfg.PauseImage = newPauseImage
+		cfg.PauseImage = config.DefaultPauseImage
```

Comment:
The man pages anyway said 3.2 prior.  Do we need to also check from 3.2 here?

Reply:
Check both 3.1 and 3.2?
Kubernetes 1.17 by default uses 3.1.